cache/local.go

```diff
@@ -0,0 +1,111 @@
+//   Copyright 2019 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package cache
+
+import (
+	"crypto/md5"
+	"database/sql"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/trackit/jsonlog"
+
+	"github.com/trackit/trackit-server/db"
+	"github.com/trackit/trackit-server/models"
+	"github.com/trackit/trackit-server/routes"
+	"github.com/trackit/trackit-server/users"
+)
+
+// formatKey is unique depending on user's AWS' identities (personal + shared accounts)
+// or identities passed in arguments and route's URL
+func formatKey(rdCache *redisCache) {
+	rdCache.key = fmt.Sprintf("%x-%x:", md5.Sum([]byte(rdCache.route)), md5.Sum([]byte(rdCache.args)))
+	for _, val := range rdCache.awsAccount {
+		rdCache.key = fmt.Sprintf("%v%v:", rdCache.key, val)
```

Comment:
Why not use `%v-%v` to have a key that would look like `XXXX-XXXX-XXXX` ?

Reply:
You right, it is a better aesthetic.